src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -568,7 +621,10 @@ private Optional<Path> inferPlatformIncludePath() {
                 }
             }
         }
-
         return Optional.empty();
     }
+
+    private String parseFrameworkPath(String optionString) {
```

Comment:
Maybe rename to `format...` since this is not parsing anything?
```suggestion
    private String formatFrameworkPath(String optionString) {
```